src/main/java/logic/CalculationCommand.java

```diff
@@ -0,0 +1,47 @@
+package logic;
+
+import calculation.CalculationLogic;
+import org.javacord.api.entity.message.Message;
+import org.javacord.api.entity.user.User;
+import org.javacord.api.event.message.MessageCreateEvent;
+import org.javacord.api.interaction.SlashCommandInteraction;
+import org.javacord.api.interaction.SlashCommandInteractionOption;
+import org.javacord.api.interaction.SlashCommandOption;
+import org.javacord.api.interaction.SlashCommandOptionType;
+import org.javacord.api.interaction.callback.InteractionImmediateResponseBuilder;
+import pw.mihou.velen.interfaces.*;
+
+import java.util.List;
+
+public class CalculationCommand implements VelenEvent, VelenSlashEvent {
+    @Override
+    public void onEvent(MessageCreateEvent event, Message message, User user, String[] args) {
+        CalculationLogic.beginCalculations(event.getApi(), event.getChannel(), Long.parseLong(args[0]), message.getId());
+    }
+
+    @Override
+    public void onEvent(SlashCommandInteraction event, User user, VelenArguments args,
+                        List<SlashCommandInteractionOption> options, InteractionImmediateResponseBuilder firstResponder) {
+
```

Comment:
Looks like there should be something here to handle slash commands. If it's text only, you can remove the class inheriting `VelenSlashEvent`

Reply:
Yes Slash Commands will be added. I just hadn't dealt with them.